docker-compose.yml

```diff
@@ -197,13 +204,21 @@ services:
   ingestion:
     image: web3labs/epirus-free-ingestion:latest
     environment:
-      - NODE_ENDPOINT=http://foundry:8541
+      - NODE_ENDPOINT=http://foundry:8545
       - MONGO_CLIENT_URI=mongodb://mongodb:27017
       - MONGO_DB_NAME=epirus
       - LIST_OF_METRICS_TO_CALCULATE_PER_MINUTE=hourly,daily,monthly,yearly
     depends_on:
       - mongodb
       - redis
       - foundry
+    networks:
+      - simulation
+
+  waku-rln-contract:
```

Comment:
are we using this? if not can we remove it?
note also that `/submodules/waku-rln-contract` is not present by default? so guess it would be needed to run "git submodule init" or similar?

in any case, if we don't use this, let's remove it. unsure if you plan to use it for future PRs, but doesn't seem related to this PR.

Reply:
I was trying something for the contract deployment and forgot to remove this, will fix it

Reply:
Fixed in [df14929](https://github.com/waku-org/waku-simulator/pull/23/commits/df1492978879fc7a442089f8cccc05e4fd12b559)